Justfile

```diff
@@ -147,3 +147,7 @@ test-all *ARGS: devenv build-cohort-extractor
     [[ -v CI ]] && echo "::group::Run tests (click to view)" || echo "Run tests"
     $BIN/python -m pytest --cov=cohortextractor --cov=tests {{ ARGS }}
     [[ -v CI ]]  && echo "::endgroup::" || echo ""
+
+# run the validate contract test only for all backends
+test-contracts *ARGS: devenv
```

Comment:
`ARGS` isn't being used. It's probably less useful here than elsewhere, because the most frequent use is to specify a single test, but it could be useful, so there's no harm in having it as long as it's passed on to `pytest`.

Reply:
thanks - I have removed ARGS for clarity.